x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/EnrichPolicyRunner.java

```diff
@@ -352,8 +355,39 @@ private void transferDataToEnrichIndex(final String destinationIndexName) {
             public void onResponse(BulkByScrollResponse bulkByScrollResponse) {
                 // Do we want to fail the request if there were failures during the reindex process?
                 if (bulkByScrollResponse.getBulkFailures().size() > 0) {
+                    logger.warn(
+                        "Policy [{}]: encountered [{}] bulk failures. Turn on DEBUG logging for details.",
+                        policyName,
+                        bulkByScrollResponse.getBulkFailures().size()
+                    );
+                    for (BulkItemResponse.Failure failure : bulkByScrollResponse.getBulkFailures()) {
+                        logger.debug(
+                            new ParameterizedMessage(
+                                "Policy [{}]: bulk index failed for index [{}], id [{}]",
+                                failure.getIndex(),
```

Comment:
`policyName` is missing as argument.

Reply:
fixed :D. Also wrapped in a `logger.isDebugEnabled()` as there is no point in looping over them unless debug is enabled